autoload/lsp_settings.vim

```diff
@@ -68,3 +68,12 @@ function! lsp_settings#exec_path(cmd) abort
   endif
   return ''
 endfunction
+
+function! lsp_settings#register_server_settings(settings) abort
+  let settings = a:settings
```

Comment:
If I set `call lsp#register_server(a:settings)`, following error raised.
```
"autoload/lsp_settings.vim" 79L, 2076C
If you want to enable Language Server, please do :LspInstallServer
Error detected while processing User Auto commands for "lsp_setup":
E121: Undefined variable: a:settings
Press ENTER or type command to continue
```
`autocmd` can't take `a:settings` or `l:settings`?

Reply:
autocmd can’t take local scope var, so change to script scope var